lib/openstudio-standards/standards/ashrae_90_1/ashrae_90_1_2019/data/ashrae_90_1_2019.prototype_inputs.json

```diff
@@ -2065,7 +2065,7 @@
       "building_type": "SecondarySchool",
       "hvac_system": null,
       "geometry_osm": "ASHRAESecondarySchool.osm",
-      "hvac_json": "ASHRAE90120162019SecondarySchool.json",
+      "hvac_json": "ASHRAE20162019SecondarySchool.json",
```

Comment:
#1060 used "ASHRAE90120162019SecondarySchool.json" which is what is currently on the master branch of the repository (see [here](https://github.com/NREL/openstudio-standards/blob/master/data/geometry/ASHRAE90120162019SecondarySchool.json)), not sure if this is a valid change.